src/components/Media/Media.js

```diff
@@ -1,7 +1,22 @@
 import './Media.css';
 
-const Media = () => {
-    
+const Media = ({videos}) => {
+    const videoKeys = videos.filter(video => video.type !== 'Trailer').map(video => video.key)
+    const trailer = videos.find(video => video.type === "Trailer")
+
+    return (
+        <section className='Media'>
+            <div >
+                {trailer && <iframe className='Main-video' src={`https://www.youtube.com/embed/${trailer.key}`} title="YouTube video player" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowFullScreen></iframe>}
+            </div>
```

Comment:
This is really cool! Did you put all the allow properties in or is that what came with the suggested code from YouTube? I'm gonna google what they all do!